src/gui_tooltips.h

```diff
@@ -37,8 +37,7 @@ enum TooltipFlags {
 };
 
 /******************************************************************************/
-DLLIMPORT extern long _DK_tooltip_scroll_offset;
-#define tooltip_scroll_offset _DK_tooltip_scroll_offset
+float tooltip_scroll_offset;
```

Comment:
you don't want the actual definition of the var inside of the header file, you'll want to move this to the .c file and if it were used in multiple c files do it in 1 and add and extern in the .h, but here it's only used in gui_tooltips.c so no need for it to be in any header